### PR TITLE
Sos 394 make savings from aws a negative cost

### PR DESCRIPTION
**Pull request by chrisperez626**
## Why are these changes needed?
As a real AWS user, I will have various product and use based discounts applied to my account. I want those to be reflected in my spend accordingly, so they should reduce the appropriate spend, so that I have a clear picture of my spend.

 

## Related issue number
[SOS-394](https://nichols.atlassian.net/browse/SOS-394?atlOrigin=eyJpIjoiOWU4OTQxMmFkYTc2NDEwZTljM2Y1ODBkZTExMjVkODEiLCJwIjoiaiJ9)

## Checks

- [X] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [ ] I've made sure all auto checks have passed.

[SOS-394]: https://nichols.atlassian.net/browse/SOS-394?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: tests/aws_conf.yaml [hunk 1]
```diff
@@ -7,4 +7,4 @@ modules:
       - provider: aws
         start: "2023-08-01"
         end: "2024-03-01"
-        account: "foo" #Account with no profile
```
Comment (aylusltd) [inline tests/aws_conf.yaml:10]: that profile exists on the machine

File: src/verinfast/cloud/aws/costs.py [hunk 2]
```diff
@@ -4,6 +4,7 @@
 
 from verinfast.utils.utils import DebugLog
 from verinfast.cloud.aws.get_profile import find_profile
+
```
Comment (aylusltd) [inline src/verinfast/cloud/aws/costs.py:7]: no whitespace changes please

File: tests/test_aws.py [hunk 3]
```diff
@@ -43,6 +43,9 @@ def test_aws_scan(self):
     with open(results_dir.joinpath(f"aws-cost-{sub_id}.json")) as f:
         costs = json.load(f)
         assert len(costs["data"]) >= 100
+        for cost in costs["data"]:
```
Comment (aylusltd) [inline tests/test_aws.py:46]: use a fixture for this test. I don't think we have a savings plan with AWS

**Review by aylusltd — CHANGES_REQUESTED** [review]

File: tests/test_aws.py (hunk 3 above)
Comment (chrisperez626) [reply tests/test_aws.py:46]: What should I use as the fixture? is there an `aws-cost-foo.json` that I can use?

**Pull request closed without merging**